src/richie/apps/core/views/error.py

```diff
@@ -0,0 +1,64 @@
+"""Error views"""
+from django.shortcuts import render
+from django.utils.translation import gettext_lazy as _
+
+CONTEXT_ERRORS = {
+    400: {
+        "title": _("Bad request"),
+        "content": _("Something went wrong. Please retry later."),
+    },
+    403: {
+        "title": _("Forbidden"),
+        "content": _(
+            "You do not have required permissions to access to this resource."
+        ),
+    },
+    404: {
+        "title": _("Page not found"),
+        "content": _("The requested resource does not exists."),
+    },
+    500: {
+        "title": _("Server bad request"),
+        "content": _("Something went wrong. Please retry later."),
```

Comment:
I think retry is used in IT only.
```suggestion
        "content": _("Something went wrong. Please try again later."),
```